test/styled/tree.test.ts

```diff
@@ -15,11 +15,10 @@ describe('styled/tree', () => {
       tree.nodes.bar.insert('baz', subtree)
 
       tree.display()
-      expect(output.stdout).to.equal(`├─ foo
-└─ bar
-   └─ baz
-      └─ qux
-
+      expect(output.stdout).to.equal(`├─ bar
+│  └─ baz
+│     └─ qux
+└─ foo
 `)
```

Comment:
It appears that before there were two line breaks added: one by the logger and one by the treeify. So that is now fixed as well.

Reply:
@simlu How do I disable auto-sorting? I want to add an api to turn it on.

Reply:
I'm not sure turning off auto sorting makes a lot of sense since js object keys are not sorted (so the order would be random). But you could just pass in a function that always return zero if you want that behavior:

`softFn = () => 0`

Reply:
Insertion order is preserved in ES6 (non-integer keys).

Reply:
Interesting! I didn't know that. Makes sense to disable sorting by default then. I'll push an update to object-treeify later.

Reply:
@RasPhilCo I've now also updated this pr.